internal/client/githubclient/githubclient_test.go

```diff
@@ -437,11 +437,11 @@ func Test_githubClient_GetWorkflow(t *testing.T) {
 				return
 			}
 			if len(got) != len(test.want) {
-				t.Fatalf("GetWorkflow got %v, want %v", got, test.want)
+				t.Fatalf("GetWorkflow() got = %v, want %v", err, test.want)
```

Comment:
why is this set to `err` should it not be `got`?

Reply:
should be length does not match error

Reply:
Oh, I forgot to change this during rebase. Thanks!

Reply:
Updated to a length not matching error.